org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/manifest.xml

```diff
@@ -3,9 +3,9 @@
   <!-- associated voice handler -->
   <voice id="" />
   <!-- associated schema -->
-  <schema id="org.OpenT2T.Thermostat.SuperPopular.Sample" />
+  <schema id="org.opent2t.sample.thermostat.superpopular" />
   <!-- associated onboarding module -->
```

Comment:
Are we intending to support both a single <schema> element at the top level as well as one or more <schema> elements under a <schemas> element? The parsing code in my lib PR currently only supports the latter, but I can change it.